metrics/micrometer-metrics/pom.xml

```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>io.dropwizard.metrics</groupId>
             <artifactId>metrics-core</artifactId>
-            <version>4.1.2</version>
+            <version>4.2.7</version>
```

Comment:
please use {metrics.version} in this <version></version>

Reply:
@stormbroken pls take a look?